<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesBoxClass</class>
 <widget class="QDialog" name="PreferencesBoxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="toolTip">
        <string/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Language of user interface</string>
         </property>
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>121</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Language of user interface&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(use &lt;span style=&quot; font-style:italic;&quot;&gt;AUTO&lt;/span&gt; to use system language)&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Auto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="en">English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="de">German</string>
          </property>
         </item>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>90</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Start discovery of remote devices when
program starts - otherwise start discovery
manually with toolbar button.</string>
         </property>
         <property name="text">
          <string>Start device discovery on startup</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Medion MD86097</string>
        </attribute>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Enable this type of device - 
discovery service will try to find
remote devices of this type.</string>
         </property>
         <property name="text">
          <string>Enable protocol</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>151</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Specify method of device discovery:
&lt;ul&gt;
&lt;li&gt;&lt;em&gt;Broadcast local&lt;/em&gt; &lt;b&gt;--&lt;/b&gt; use broadcast messages in local network (specified by network configuration) (&lt;b&gt;default&lt;/b&gt;)&lt;/li&gt;
&lt;li&gt;&lt;em&gt;Broadcast global&lt;/em&gt; &lt;b&gt;--&lt;/b&gt; use broadcast messages (simple broadcast to all devices)&lt;/li&gt;
&lt;li&gt;&lt;em&gt;Selected IP addresses&lt;/em&gt; &lt;b&gt;--&lt;/b&gt; try to find devices on given IP addresses in network&lt;/li&gt;
&lt;/ul&gt;
&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Broadcast local</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string extracomment="asd">Broadcast global/simple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Selected IP address(es)</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Run discovery on selected address type</string>
         </property>
         <property name="text">
          <string>Discovery address type:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>70</y>
           <width>251</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>IP addresses of USB hub devices (separated by comma)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Discovery port:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>71</width>
           <height>25</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>16708</string>
         </property>
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Control connection port:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>150</y>
           <width>71</width>
           <height>25</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>21827</string>
         </property>
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>301</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Include username additional to hostname &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;in disconnect request - like &lt;span style=&quot; font-style:italic;&quot;&gt;name@host&lt;/span&gt;.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(for devices connected to different hosts)&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Include username in disconnect request</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Extra / Debug</string>
        </attribute>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Write log / debug messages to console</string>
         </property>
         <property name="text">
          <string>Console output</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Logging:</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>80</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Write log / debug messages to file(s)</string>
         </property>
         <property name="text">
          <string>File output</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>80</y>
           <width>261</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Directory for log files
(will be created if not existing)
[use / as separator!]</string>
         </property>
         <property name="text">
          <string>/tmp/USBhubConnect/</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>80</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Dir:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>75</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string notr="true">ERROR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">WARN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">INFO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">DEBUG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">TRACE</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>140</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Log level:</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>110</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Append to existing files</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesBoxClass</receiver>
   <slot>okButtonSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesBoxClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okButtonSlot()</slot>
 </slots>
</ui>
